src/test/kotlin/step3/racing_car/RacingCarTest.kt

```diff
@@ -0,0 +1,72 @@
+package step3.racing_car
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+import step3.racing_car.ui.InputView
+
+/**
+ * 자동차 경주 단위 테스트 클래스
+ *
+ * @author KimJungSik
+ * @since 2023/10/28
+ * */
+class RacingCarTest: FunSpec({
+
+    test("자동차 대수나 횟수를 입력하지 않은 경우 에러가 발생하는지 확인한다") {
+        RacingCar.errorCheck("")
+    }
+
+    test("자동차 대수나 횟수에 문자를 입력하는 경우 에러가 발생하는지 확인한다") {
```

Comment:
xxx 확인한다는건 에러가 나는게 정상인지 안나는게 정상인지 가늠하기 어려운 설명이에요. 
보통 저는 테스트를 작성할때 description은 
test하려는 기능 / 테스트 조건 / 유추되는 결과로 요약하는 편입니다. 

해당 테스트같은 경우 저라면 다음과 같이 하는 편인데 참고하시면 좋을 것 같아요.
```suggestion
    test("자동차 생성 파라미터에 유효하지 않은 문자열을 전달할 경우 예외를 던진다.") {
```